parm/config/config.fcst

```diff
@@ -155,7 +155,14 @@ if [[ "$OUTPUT_FILE" == "netcdf" ]]; then
     export  ichunk2d=0; export jchunk2d=0
     export  ichunk3d=0; export jchunk3d=0;  export kchunk3d=0
     RESTILE=`echo $CASE |cut -c 2-`
-    if [[ "$machine" == "WCOSS_DELL_P3" ]] || [[ "$machine" == "WCOSS2" ]]; then
+    if [[ "$machine" == "WCOSS2" ]]; then
```

Comment:
same here.

Reply:
cleaned up and consolidated, please see updates